frontend/src/Calculator.tsx

```diff
@@ -60,6 +69,19 @@ export default function ProductionHelper() {
                             setCurrentFleetSupply={setCurrentFleetSupply}
                             maxFleetSupply={maxFleetSupply}
                             setMaxFleetSupply={setMaxFleetSupply}
+                            shipCapacityUsed={shipCapacityUsed}
+                            setShipCapacityUsed={setShipCapacityUsed}
+                            maxShipCapacity={maxShipCapacity}
+                            setMaxShipCapacity={setMaxShipCapacity}
+                            spaceDockFighterBonus={spaceDockFighterBonus}
+                            setSpaceDockFighterBonus={setSpaceDockFighterBonus}
+                        />
+                        <Checkbox
+                            label="Fighter II"
+                            checked={isFighterUpgraded}
+                            onChange={() =>
+                                setIsFighterUpgraded(!isFighterUpgraded)
+                            }
```

Comment:
I was on the fence about putting the checkbox inside `BudgetModeInputs` rather than adjacent, but I could foresee wanting to use the checkbox in both modes if we get fancier, and putting it here makes it easier to move around if that happens. Not married to it either way though!

Reply:
I think I lean (slightly) the other way, but that's because I only really imagine using this in the budget mode. Doesn't really make a difference to me as this is easy to refactor if we guess wrong.

:thought_balloon: One thing I am considering though - I think with all this functionality, budget mode is getting a little complex. I *like* the functionality, but it's a lot of paperwork to do. What would you think about (in the future, not this PR) hiding some of the more complex options like capacity management or off-planet (read: Saar) production, under a dropdown tab for "advanced mode"? That way the options are there for anyone who wants to deal with them, and hidden (+inactive) for anyone who can't be bothered?

Reply:
Another example in support of an "advanced mode" - isn't the "Space Dock Fighter Bonus" the same value for *almost* every faction?

Reply:
Done, moved firmly into budget mode 👌 

That's a great idea, all the form fields are making my eyes glaze over!

Reply:
Oh dang didn't know that about the space dock bonus - the only two factions I've played have been different so I just assumed it varied! Should definitely make that a static number which you can change in advanced mode